src/elements/content-uploader/ProgressBar.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import './ProgressBar.scss';
+
+type Props = {
+    percent?: number;
+};
+
+const ProgressBar = ({ percent }: Props) => {
+    const [currentPercent, setCurrentPercent] = useState(percent || 0);
+
+    useEffect(() => {
+        setCurrentPercent(percent);
+    }, [percent]);
+
+    let controlledPercent = currentPercent > 100 ? 100 : currentPercent;
```

Comment:
Can we use some `Math` operators to clamp these values and avoid multiple ternaries?

```
const clampedPercentage = Math.max(0, Math.min(100, percent))
```